cf-java-logging-support-servlet/src/main/java/com/sap/hcp/cf/logging/servlet/filter/CorrelationIdFilter.java

```diff
@@ -59,29 +56,31 @@ private String determineCorrelationId(HttpServletRequest request) {
             correlationId = String.valueOf(UUID.randomUUID());
             // add correlation-id as custom field, since it is added to MDC only
             // in the next step
-            LOG.debug("Generated new correlation-id <{}>", correlationId, customField(correlationHeader.getField(),
-                                                                                      correlationId));
+            LOG.debug("Generated new correlation-id <{}>", correlationId);
```

Comment:
Why do you think, the custom field can go? By what mechanism will it be part of this log message?

Reply:
Actually, this is a silent parameter. It is not rendered in the pattern but still picked-up by the JSONEncoder. This will render a custom field "correlation-id": "UUID" in the log message. It is the work-aroung in SLF4J 1.x, where there are no key-value-pairs available for this purpose.

Reply:
Thanks for the explanation, I'll revert this part.